### PR TITLE
Locale and FAQ Update

### PR DESCRIPTION
**Pull request by Dormant-Hero**
Locale changes and some reward shop code change just to detect if English Locale or another locale. Very small change.

The FAQ I have added a lot of content and not sure how it would look on the live website but should be plenty to work with for the FAQ's next update. Change as you see fit boss :)

### CODE REVIEW

**Comment by cipherxof** [comment]
Why isn't the FAQ using the localization system?

**Comment by Dormant-Hero** [comment]
I wanted to check the content with you first. It is a bit easier to make the changes and see their impact in the html sort of format instead of app.etc. 

If you are fine with the content and how it looks in your version in which you can see the layout better or closer to the live version of the website then I can start the English localisation right away in which case I will want to get all the other languages also so the faq to match the selected language right away.

**Comment by cipherxof** [comment]
I feel like the paragraphs don't really flow well and sound a bit like a casual conversation rather than a document / website. It maybe worth running them through ChatGPT or something.

**Comment by Dormant-Hero** [comment]
Okay so just updated the code with chatgpt's touches on making the language more professional and less language. I also specified what our website is. Hopefully this is better.

File: src/pages/FAQ.tsx [hunk 1]
```diff
@@ -2,105 +2,109 @@ import { PageContainer } from '@ant-design/pro-layout';
...
-              <li>GPU: Anything with Vulkan Support</li>
-            </ul>
+            <p>To register, please visit <a href="https://mgo2pc.com/register">our registration page</a>. We advise using a Gmail account for registration as Hotmail/Outlook emails have been experiencing issues lately. Your username should be at least 8 characters long, containing only letters and numbers, without spaces or special characters. This username will also serve as your Game ID, and the chosen password will be your game password. Once your account is created, it is not necessary to change your password, even if redirected to a password change page.</p>
+          </div>
+        </Panel>
+        <Panel header="Email Confirmation Issues" key="2">
+          <div>
+          <p>If you do not receive an email confirmation, it is possible that a Hotmail/Outlook email address was used, which might cause delays. However, account activation is not required. Please ensure you remember your username and password. If forgotten, assistance can be sought by opening a ticket in our <a href="https://discord.com/invite/MNqmzuW2wH">Discord channel</a>.</p>
+          </div>
```
Comment (cipherxof) [inline src/pages/FAQ.tsx:22]: These are the only ones that I'm not a fan of. They're too wordy, and sound a bit awkward. I don't think we need the Registration Process entry at all.

I'll merge for now and then update on my end.
